sample.env

```diff
@@ -0,0 +1,26 @@
+# Docker compose env file
+# https://docs.docker.com/compose/env-file/
+# https://docs.docker.com/compose/reference/envvars/#compose_file
+COMPOSE_FILE=docker-compose.yml
+COMPOSE_PROJECT_NAME=kaiyuanshe_osschat
+
+#WEBHOOK_PROXY_URL=https://smee.io/pAaHJ4MtZN0K8fb
+#APP_ID=
+#PRIVATE_KEY=
+#WEBHOOK_SECRET=
+
+# Wechaty
+HA_WECHATY_PUPPET=
+HA_WECHATY_PUPPET_TOKEN=
+CHATOPS_ROOM_ID=
+HEARTBEAT_ROOM_ID=
+DEVELOPER_ROOM_ID=
+
+# GitHub APP 
+GITHUB_CLIENT_ID=
+GITHUB_CLIENT_SECRET=
+
+# Chatopera BOT Services to handle requests
+WCP_DEFAULT_CLIENTID=
```

Comment:
What is `WCP` mean?

I'd like to rename them with the prefix `CHATOPERA_XXX`

Reply:
WCP is short for Wechaty Chatopera Plugin, to make the ENV names short and catchy.